projects/plugins/jetpack/modules/sharedaddy/sharing.php

```diff
@@ -29,7 +29,15 @@ public function __construct() {
 		require_once WP_SHARING_PLUGIN_DIR . 'sharing-service.php';
 
 		add_action( 'admin_init', array( $this, 'admin_init' ) );
-		add_action( 'admin_menu', array( $this, 'subscription_menu' ) );
+
+		/**
+		 * The following action is commented out since it creates an entry to a legacy page.
+		 *
+		 * This menu item was previously hidden via https://github.com/Automattic/jetpack/pull/1960
+		 * but now we are preventing its registration altogether.
+		 *
+		 * add_action( 'admin_menu', array( $this, 'subscription_menu' ) );
```

Comment:
My thinking is that commenting out the code and leaving a note would provide more context of this change, and minimize the "blast radius" of the change.

Alternatively we can simply remove this line and remove the function `subscription_menu`.